astropy/utils/masked/tests/test_function_helpers.py

```diff
@@ -579,6 +579,12 @@ def check(self, function, *args, method=None, **kwargs):
         x = getattr(self.ma, method)(*args, **kwargs)
         assert_masked_equal(o, x)
 
+    def test_max(self):
```

Comment:
Just in case you're curious, for `Masked` all functions in `fromnumeric.py` are included automatically as subclass safe functions, hence unlike for `Quantity`, there was no need to add them explicitly.